docs/blog/content/material/ElasticJob.en.md

```diff
@@ -1,26 +1,37 @@
 
-Since ElasticJob has been open sourced at the end of 2014, it has been more than 5 years. With its rich features, comprehensive documentation, high-quality code and ease of use of the framework, it has accumulated a large number of loyal users and a good industry reputation (5.8K star). It has been one of the most popular projects in the field of distributed scheduling framework.
+ * Task aggregation
+ * Dynamic resource scheduling
+ * Fault detection and automatic repairing
+ * Failover and retry
+ * Full-featured operation platform and management tools
+ * Friendly with cloud-native
 
-In the past two years, ElasticJob has slowed down its development due to core developers' efforts to develop and maintain the Apache ShardingSphere project. However, with the continuous expansion of the Apache ShardingSphere project in the distributed field and the user demands for functions such as distributed management of multiple data cluster environments and elastic expansion of data migration, a powerful distributed scheduling component is required. As a result, the Apache ShardingSphere community has planned to initiate a proposal to use ElasticJob as its subproject for continuous update and maintenance.
+ ElasticJob can fully meet enterprise user's requirements on scheduling and processing ability for task management and batch jobs.
 
-We made this good news public for the first time and we will bring more information about ElasticJob in the future.
+<!-- ElasticJob自2014年底开源以来，经历了5年多的发展，以其功能的丰富性，文档的全面性，代码的高质量，框架的易用性，积累了大量的忠实用户和良好的业内口碑（5.8K star），一直也是分布式调度框架领域最受大家欢迎的项目之一。 -->
+ElasticJob began to open it's source code at end of 2014. Since richness functions,quite completed documents and simplicity to use,it earns a lot of loyal users and good word of mouth in the past five years. ElasticJob is one of the most popular distributed scheduling framwork and gains more than 5.8K star on github.
 
-> Discussion Email：
-https://lists.apache.org/thread.html/rd6171e2065be6bcfbeb7aba7e5c876eeed04db585c6ab78fc03a581c%40%3Cdev.shardingsphere.apache.org%3E
+<!-- 近两年来，由于核心开发者全力发展和维护Apache ShardingSphere项目等原因，ElasticJob放缓了发展的脚步。但是随着Apache ShardingSphere项目在分布式领域的不断拓展，以及用户对于多数据集群环境的分布式管理和数据迁移的弹性扩容等功能，需要一个强大的分布式调度组件。由此为契机，Apache ShardingSphere社区已经计划发起提议将ElasticJob作为其子项目进行长期持续的更新和维护。 -->
+In the last two years, the core developer spent much time to maintain and contribute to Apache ShardingSphere have caused ElasticJob development slowly.But with the developement of Apache ShardingSphere on distributed domain,we need a distributed scheduling component to support distributed management of multi cluster,data migration and scaling. So Apache ShardingSphere community have put up a proposal,make ElasticJob as a sub project,that makes sure ElasticJob could be maintain and update in the future.
```

Comment:
> the core developer **spent** much time to maintain and contribute to Apache ShardingSphere **have** caused ElasticJob development slowly.
> So Apache ShardingSphere community have put up a proposal,**make** ElasticJob as a sub project,that makes sure ElasticJob could be **maintain** and **update** in the future.

Hi @rudy2steiner Thanks for your translation and welcome!
I just gave it a glance and found some bad grammar, which made those sentences hard understood. So IMO, it is needed to check the grammar and word spelling with the help of APPs, like MS word.

Reply:
Thanks, any more Grammarly issue?